scripts/build-packages.js

```diff
@@ -0,0 +1,26 @@
+const path = require("path");
+const fs = require("fs");
+const shell = require("shelljs");
+
+shell.cd(path.join(__dirname, ".."));
+
+const packages = fs
+    .readdirSync(path.join(__dirname, "..", "packages"))
+    .filter(name => !name.startsWith("."));
+
+for (const package of packages) {
+    const pkgPath = path.join(__dirname, "..", "packages", package);
+    const pkgJSONPath = path.join(pkgPath, "package.json");
+
+    if (fs.existsSync(pkgJSONPath)) {
+        const pkgJSON = require(pkgJSONPath);
+
+        if (pkgJSON && pkgJSON.scripts && pkgJSON.scripts.build) {
+            console.log(`\nbuilding "/packages/${package}"\n`);
+
+            if (shell.exec(`yarn --cwd "${pkgPath}" build`).code !== 0) {
+                shell.exit(1);
+            }
+        }
+    }
```

Comment:
```suggestion
    console.log(`\nbuilding "/packages/${package}"\n`);

    const buildExitCode = shell.exec(`yarn workspace ${package} lint`).code;

    if (buildExitCode !== 0) {
        shell.exit(buildExitCode.code);
    }
```

Reply:
Would this build `desktop` and `web` too?

Reply:
There will be a package `flat-i18n` which does not have a `build` script.

Reply:
The previous code did not consider this situation either.

Reply:
> The previous code did not consider this situation either.

Sorry I misread the code before. Comment updated.

Reply:
OK, It’s better to add a comment on line 18 to explain the purpose

Reply:
Done.